packages/syft/src/syft/core/node/common/node_manager/user_manager.py

```diff
@@ -259,47 +301,67 @@ def set(  # nosec
         self.modify({"id": user_id}, {key: value})
 
     def can_create_users(self, verify_key: VerifyKey) -> bool:
+        """Checks if a user has permissions to create new users."""
```

Comment:
Q: should we include the args (verify_key) and error (UserNotFoundError) in the docstring?, I know it might be higly reptitive in the subsequent functions.?